commands/theme/preview.ts

```diff
@@ -29,7 +29,7 @@ const { preview } = require('@hubspot/theme-preview-dev-server');
 const { hasFeature } = require('../../lib/hasFeature');
 const i18nKey = 'commands.theme.subcommands.preview';
 
-exports.command = 'preview [--src] [--dest]';
+exports.command = 'preview [--path] [--dest]';
```

Comment:
iirc we had opted to keep src and dest whenever both are present, and then use `path` instead of `src` if it's the only input.

Reply:
Revert the changes https://github.com/HubSpot/hubspot-cli/pull/1270/commits/331c932ccceb905beb3c917bd2eb84d4e6cfea57